### DISCUSSION TITLE
Linux çekirdeği nedir?

### DISCUSSION THREAD

**Discussion by testacc123co** [enzifiri/GitHub-Achievements#509] [Q&A] [answered]
Linux çekirdeği nedir?

**Comment by enzifiri** [comment, marked as answer]
test

**Comment by codelovingcat** [comment]
Kernel işletim sisteminin çekirdeğidir. İşletim sistemleri ise donanımı yöneten ve kullanıcıların bilgisayar üzerinde programlarını çalıştırabilmelerine izin veren uygulamaların tümüdür.
